spec/unit/application_spec.rb

```diff
@@ -142,19 +142,28 @@
 
     it "should yield a new namespace" do
       application.namespace :new_namespace do |ns|
-        expect(ns.name).to eq :new_namespace
+        ActiveSupport::Deprecation.silence do
+          expect(ns.name).to eq :new_namespace
+          expect(ns.name_path).to eq [:admin, :new_namespace]
```

Comment:
Minor nitpick, but `expect(ns.name_path).to eq [:admin, :new_namespace]` can be outside `Deprecation.silence` block.